### PR TITLE
Add films_with_biggest_crew, films_in_each_language, most_popular_directors, directors_with_biggest_amount_of_genres records

### PR DESCRIPTION
**Pull request by DanyloSarvasNulp**

### CODE REVIEW

**Review by velychkovych — CHANGES_REQUESTED** [review]
Wait until auto import branch will be merged, merge main into your branch and update this PR.

File: reports.py
Comment (velychkovych) [inline reports.py:409]: Here the same as comment above.

File: reports.py
Comment (velychkovych) [inline reports.py]: Here the same as comment above.

File: reports.py
Comment (velychkovych) [inline reports.py]: I would add an alias to this column, but it's up to you.

**Review by velychkovych — CHANGES_REQUESTED** [review]

**Review by C0l1ns — APPROVED** [review]

**Pull request merged**
